cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -419,35 +422,39 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
     #------------------------------------------------------------------------------
     # Remove exported OpenMP flags because they are not language agnostic
     #------------------------------------------------------------------------------
-    set(_props)
-    if( ${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13.0" )
-        list(APPEND _props INTERFACE_LINK_OPTIONS)
-    endif()
-    list(APPEND _props INTERFACE_COMPILE_OPTIONS)
+    if (NOT SERAC_ENABLE_CODEVELOP)
```

Comment:
Not connected to this PR but ....

Sigh this section of code makes me so sad... I am hopefully revamping all the BLT targets to get rid of this.